modules/openapi-generator/src/main/resources/typescript-rxjs/runtime.mustache

```diff
@@ -167,12 +163,21 @@ const queryString = (params: HttpQuery): string => Object.keys(params)
 // alias fallback for not being a breaking change
 export const querystring = queryString;
 
+/**
+ * @deprecated
+ */
 export const throwIfRequired = (params: {[key: string]: any}, key: string, nickname: string) => {
-    if (!params || params[key] === null || params[key] === undefined) {
+    if (!params || params[key] == null) {
         throw new RequiredError(`Required parameter ${key} was null or undefined when calling ${nickname}.`);
     }
 };
 
+export const throwIfNullOrUndefined = (value: any, nickname?: string) => {
+    if (value == null) {
+        throw new Error(`Parameter "${value}" was null or undefined when calling "${nickname}".`);
```

Comment:
Won't this here  just print `Parameter <null/undefined> was null or undefined when calling <method>`? Previously, `throwIfRequired` passed in the param name as a string, but here we are only printing the value.

Reply:
@jtvd78 would you like to file a PR to fix this?

Reply:
I can do that

Reply:
See https://github.com/OpenAPITools/openapi-generator/pull/6083